### DISCUSSION TITLE
Request ability to select multiple tracks when browsing

### DISCUSSION THREAD

**Discussion by querys1** [music-assistant/support#968] [Feature requests and ideas]
Hello,
any idea, how to play a folder instead of a single track?

**Comment by OzGav** [comment]
Not sure about that as media players typically don’t work in terms of folders. Please refer to here https://github.com/music-assistant/hass-music-assistant/discussions/820 for how to organise your music folders and properly tag your music.

**Reply by querys1** [reply]
unfortunately not everything is properly taged correctly. But to be honest, I just use my folder/file structure on any device.
To be honest, for radio plays and sampler the taging is quite complicated. 
Furthermore I do not want to go through around 60k+ MP3 files and check the Tags.

So the folder view works great with MA, but I'm not able to select all files at once and add them all to a playlist or add a whole folder with one click.

**Comment by OzGav** [comment]
I am away from my instance with the moment and I don’t usually use browse. There is an issue which affects  what I am about to say https://github.com/music-assistant/hass-music-assistant/issues/821 but can’t you browse, and in list view select the tracks you want and then does the action menu option appear so you can add to playlist?

**Comment by querys1** [comment]
What ist "list view"? My MA looks like this. I do not have any option to select anything. The only possibility is to click on the track and than add it. But this is quite uncomfortable, when you have anything with a lot of tracks.
![view](https://user-images.githubusercontent.com/116440848/198831355-46357b58-3bdc-4318-a456-0080a6472881.jpg)

**Reply by querys1** [reply]
Ah I see, when I brows through Artist or Album etc. I have the possibility to add more than one. This Tick-Box is missing for the browser-view. Maybe this could be added?
![Album-view](https://user-images.githubusercontent.com/116440848/198832137-37eb2b7c-3a48-417c-a41c-d2d357c83d91.jpg)

**Reply by OzGav** [reply]
Yeah I wasn’t sure. I will change this to a feature request.

**Reply by querys1** [reply]
> Yeah I wasn’t sure. I will change this to a feature request.

this would be great and would make this for me very usefull. Maybe the whole bar could be added (with favorits, refresh, order, search and the list/raster view?

**Reply by OzGav** [reply]
Will see what others think. As I said this isn’t the usual use case for a media player….

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
